ml_model/src/main/java/edu/wm/sealab/featureextraction/FeatureVisitor.java

```diff
@@ -112,21 +127,28 @@ public void visit(AssignExpr assignExpr, Void arg) {
   }
 
   /**
-   * This method computes # comparisons in a java method
+   * This method computes # comparisons and operators in a java method
```

Comment:
"comparisons" are also a kind of operator, so this javadoc is ambiguous. It should probably say "# of comparison operators and # of arithmetic operators", or something equally specific.

Reply:
The issue requested that I find the avg number of 'operators' - I later changed it to arithmetic operators but missed this one. Thanks!